<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDialog</class>
 <widget class="QDialog" name="UserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>291</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="FirstNameEdit">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 1px solid #CFCFCF; /* A light grey border */
    border-radius: 10px;       /* Adjust this value for desired roundness */
    padding: 5px;  
color: rgb(62, 47, 28);            /* Add some internal padding */
}

QLabel {
    color: #555555; /* A darker grey */
}</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>First Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LastNameEdit">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 1px solid #CFCFCF; /* A light grey border */
    border-radius: 10px;       /* Adjust this value for desired roundness */
    padding: 5px;              /* Add some internal padding */
}
</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Last Name</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="EmailEdit">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 1px solid #CFCFCF; /* A light grey border */
    border-radius: 10px;       /* Adjust this value for desired roundness */
    padding: 5px;  
color: rgb(62, 47, 28);            /* Add some internal padding */
}

QLabel {
    color: #555555; /* A darker grey */
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>EMAIL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="UniversityEdit">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 1px solid #CFCFCF; /* A light grey border */
    border-radius: 10px;       /* Adjust this value for desired roundness */
    padding: 5px;              /* Add some internal padding */
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>UNIVERSITY</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLineEdit" name="IDedit">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 1px solid #CFCFCF; /* A light grey border */
    border-radius: 10px;       /* Adjust this value for desired roundness */
    padding: 5px;              /* Add some internal padding */
}
</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>ID CARD</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">QLabel
{
background :rgb(255, 255, 255);
 border-radius: 10px; 
}</string>
        </property>
        <property name="text">
         <string>Date Of Birth</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="DateEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="PasswordEdit">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 1px solid #CFCFCF; /* A light grey border */
    border-radius: 10px;       /* Adjust this value for desired roundness */
    padding: 5px;              /* Add some internal padding */
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="RFIDEdit">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 1px solid #CFCFCF; /* A light grey border */
    border-radius: 10px;       /* Adjust this value for desired roundness */
    padding: 5px;              /* Add some internal padding */
}
</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>RFID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="TypeComboBox">
        <property name="placeholderText">
         <string>TYPE</string>
        </property>
        <item>
         <property name="text">
          <string>student</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>teacher</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>administrator</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>librarian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>externel</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Library Director</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
